CODING_STANDARD.md

```diff
@@ -194,6 +194,12 @@ Formatting is enforced using clang-format. For more information about this, see
   which safely manages the pointer. As such, `new` should only be used in
   constructors, and `delete` in destructors. Never use `malloc` or `free`.
 
+# CProver conventions
+- Avoid if at all possible using irept methods like `get(ID_name)`, instead cast
+  to a derived type (e.g. `class_typet`) and use the wrapper method `get_name`
+- Use `can_cast_type`/`can_cast_exprt` instead of directly checking the `id()`
```

Comment:
Did you mean `can_cast_expr` (without the 't')?

Reply:
Good spot!